### PR TITLE
application: serial_lte_modem: Add secondary PDN support to #XPING

### PR DESCRIPTION
**Pull request by junqingzou**
Allow to specify which PDN to use in Ping test.
Prepared for NIDD test in the future.

Signed-off-by: Jun Qing Zou <jun.qing.zou@nordicsemi.no>

### CODE REVIEW

**Review by rlubos — APPROVED** [review]

File: applications/serial_lte_modem/doc/ICMP_AT_commands.rst [hunk 1]
```diff
@@ -38,6 +38,9 @@ Syntax
 * The ``<interval>`` parameter is an integer.
   It represents the time to wait for sending the next echo request, in milliseconds.
   Its default value is ``1000``.
+* The ``<pdn>`` parameter is an integer.
+  It represents ``cid`` in ``+CGDCONT`` command.
```
Comment (peknis) [inline applications/serial_lte_modem/doc/ICMP_AT_commands.rst:42]:
```suggestion
  It represents ``cid`` in  the ``+CGDCONT`` command.
```

File: applications/serial_lte_modem/src/slm_util.h [hunk 2]
```diff
@@ -94,12 +94,13 @@ int slm_util_atoh(const char *ascii, uint16_t ascii_len, uint8_t *hex, uint16_t
 int util_string_get(const struct at_param_list *list, size_t index, char *value, size_t *len);
 
 /**
- * @brief use AT command to get IPv4 and IPv6 addresses
+ * @brief use AT command to get IPv4 and IPv6 addresses for specified PDN
  *
+ * @param[in] cid PDP Context id as defined in "+CGDCONT" command (0~10)
```
Comment (peknis) [inline applications/serial_lte_modem/src/slm_util.h:99]:
```suggestion
 * @param[in] cid PDP Context ID as defined in "+CGDCONT" command (0~10)
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: applications/serial_lte_modem/doc/ICMP_AT_commands.rst (hunk 1 above)
Comment (junqingzou) [reply applications/serial_lte_modem/doc/ICMP_AT_commands.rst]: Applied, thanks

File: applications/serial_lte_modem/src/slm_util.h (hunk 2 above)
Comment (junqingzou) [reply applications/serial_lte_modem/src/slm_util.h]: Applied, thanks

**Review by peknis — APPROVED** [review]

**Review by melwee01 — APPROVED** [review]

**Pull request merged**